tests/conftest.py

```diff
@@ -44,9 +46,10 @@ def entry_xml(self):
               </content>
             </entry>
         """
-        self.attributes["event_date_time"] = xsDateTime_format(
-            self.attributes["event_date_time"]
-        )
+        if isinstance(self.attributes['event_date_time'], datetime.datetime):
+            self.attributes["event_date_time"] = xsDateTime_format(
+                self.attributes["event_date_time"]
+            )
```

Comment:
I know this repo mixes quotes. I don't expect you to go through the files and change them. On new changes, it makes sense to me to be consistent at least within the changeset. So, if line 49 used double quotes like in 50 and 51, I wouldn't have mentioned it.